recipes/sparrow/all/conanfile.py

```diff
@@ -0,0 +1,112 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build.cppstd import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=2.0"
+
+
+class SparrowRecipe(ConanFile):
+    name = "sparrow"
+    description = "C++20 idiomatic APIs for the Apache Arrow Columnar Format"
+    license = "Apache-2.0"
+    author = "Man Group"
```

Comment:
`author` field is generally left unset on CCI.

Reply:
Ok adressed